compose/sidam.yml

```diff
@@ -22,7 +22,7 @@ services:
     image: hmctsprivate.azurecr.io/idam/shared-db:latest
   smtp-server:
     image: mailhog/mailhog
-    scale: 0
+    scale: 1
```

Comment:
one of required test is about adding family man number. This test requires mail server to be run